src/Features/Core/Portable/SimplifyInterpolation/AbstractSimplifyInterpolationDiagnosticAnalyzer.cs

```diff
@@ -63,18 +64,16 @@ private void AnalyzeInterpolation(OperationAnalysisContext context)
                 return;
             }
 
-            var locations = ImmutableArray.Create(interpolation.Syntax.GetLocation());
+            context.ReportDiagnostic(DiagnosticHelper.Create(
+                UnnecessaryWithSuggestionDescriptor,
+                unnecessaryLocations.First(),
+                option.Notification.Severity,
+                additionalLocations: ImmutableArray.Create(interpolation.Syntax.GetLocation()),
+                properties: null));
 
-            var severity = option.Notification.Severity;
-
-            for (var i = 0; i < unnecessaryLocations.Length; i++)
+            foreach (var subsequentLocation in unnecessaryLocations.Skip(1))
```

Comment:
`for (var i = 1; i < unnecessaryLocations.Length; i++)`

Reply:
I can do this but I'm reluctant without knowing why this would be preferable.

Reply:
One reason is to just avoid the linq allocation.  Second is just that it feels clearer to me.  :)